From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Bjarne Koll <lynxplay101@gmail.com>
Date: Wed, 21 Jul 2021 02:32:45 +0200
Subject: [PATCH] Gradle project changes

This commit modifies the gradle build configuration of the server
implementation to fit the ktp fork.

diff --git a/build.gradle.kts b/build.gradle.kts
index b3687f632bbf06c933a6ef04dc2236ccf3c030b8..8ccd41bc820ee186a4c5985b2361e571a757d967 100644
--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -26,11 +26,12 @@ repositories {
     }
     // Paper end
     maven("https://repo.velocitypowered.com/snapshots/") // Paper
+    maven("https://papermc.io/repo/repository/maven-public/") // KTP - add paper maven repo
 }
 
 dependencies {
-    implementation(project(":Paper-API"))
-    implementation(project(":Paper-MojangAPI"))
+    implementation(project(":KTP-API"))
+    implementation("io.papermc.paper:paper-mojangapi:1.17.1-R0.1-SNAPSHOT")
     // Paper start
     implementation("org.jline:jline-terminal-jansi:3.12.1")
     implementation("net.minecrell:terminalconsoleappender:1.2.0")
@@ -82,7 +83,7 @@ tasks.jar {
         attributes(
             "Main-Class" to "org.bukkit.craftbukkit.Main",
             "Implementation-Title" to "CraftBukkit",
-            "Implementation-Version" to "git-Paper-$implementationVersion",
+            "Implementation-Version" to "git-ktp-$implementationVersion",
             "Implementation-Vendor" to date, // Paper
             "Specification-Title" to "Bukkit",
             "Specification-Version" to project.version,
